src/components/shortcuts.js

```diff
@@ -175,6 +184,10 @@ function executeAction(card, target, action) {
         id = card.getAttribute('data-id');
     }
 
+    const itemsContainer = dom.parentWithClass(card, 'itemsContainer');
+
+    const parentId = itemsContainer.getAttribute('data-parentid');
+
     const item = getItemInfoFromCard(card);
```

Comment:
When playing a photo album from the library, the sort from the library is used. Since playing a photo album doesn't (shouldn't) queue its neighbors, we should use its sort rather than the parent sort.

At the same time, to change the sort order of an album, we have to go into it, which is not very convenient. :man_shrugging: 
```suggestion
    const item = getItemInfoFromCard(card);

    const itemsContainer = dom.parentWithClass(card, 'itemsContainer');

    const parentId = item.IsFolder ? item.Id : itemsContainer.getAttribute('data-parentid');
```

Reply:
Cool!

Looks good to me. Later today I will look into this and test it out.

Thank you